Iniciante/2764/README.md

```diff
@@ -0,0 +1,33 @@
+# Problema:   
+O seu professor gostaria de fazer um programa com as seguintes características:
+
+1. Leia uma data no formato DD/MM/AA;
+2. Imprima a data no formato MM/DD/AA;
+3. Imprima a data no formato AA/MM/DD;
+4. Imprima a data no formato DD-MM-AA.
+
+**Problema Completo**: https://www.urionlinejudge.com.br/judge/pt/problems/view/2764
+
+# Resolução:
+
+Iniciaremos a resolução deste problema declarando as variaveis que iremos utilizar, serão 3 variaveis do tipo inteiro onde iremos armazenar os valores correspondentes as datas que iremos receber como entrada
+```c
+  int dia, mes, ano;
+```
+
+Durante a nossa leitura através da função `scanf` ao deparar com um caractere "/" ele irá interromper a leitura e ira procurar por novos valores, salvando estes em cada variavel, dividindo em 3 blocos de numeros inteiros
+```c
+	scanf("%d/%d/%d", &dia, &mes, &ano);
+```
+
+Após realizada a leitura dos 3 valores inteiros iremos apenas imprimir estes seguindo a formatação proposta pelo problema
```

Comment:
```suggestion
Após realizada a leitura dos 3 valores inteiros, iremos apenas imprimi-los seguindo a formatação proposta pelo problema.
```

Reply:
Acho que aqui é legal explicar que o argumento com esse `02` significa que o número vai ser impresso necessariamente com 2 dígitos, acrescentando zero à esquerda quando for preciso

Reply:
```suggestion
Após realizada a leitura dos 3 valores inteiros, iremos apenas imprimi-los seguindo a formatação proposta pelo problema, para isso iremos utilizar `%02d`, com isso garantimos que o valor que será apresentado irá, necessariamente, possuir 2 dígitos, colocando um "0" a esquerda quando necessário
```

Reply:
```suggestion
Após realizada a leitura dos 3 valores inteiros, iremos apenas imprimi-los seguindo a formatação proposta pelo problema. Para isto, utilizamos o argumento `%02d`, garantindo que o valor que será apresentado irá, necessariamente, possuir 2 dígitos, colocando um "0" a esquerda quando necessário.
```